.vscode/settings.json

```diff
@@ -14,5 +14,6 @@
   "prettier.trailingComma": "all",
   "prettier.singleQuote": false,
   "prettier.semi": true,
-  "prettier.printWidth": 80
+  "prettier.printWidth": 80,
+  "eslint.workingDirectories": ["packages/council-tokenlist"]
```

Comment:
We open vscode at the monorepo level, but need eslint to run w/ the local config of the given app or package. We'll probably want to add the other packages here as well.